Makefile

```diff
@@ -218,14 +218,24 @@ docker-build-all: docker-build-debug docker-build-hermes
 ###############################################################################
 ###                                Linting                                  ###
 ###############################################################################
+golangci_lint_cmd=golangci-lint
+golangci_version=v1.50.1
```

Comment:
No because it requires go `1.19`. If you bump it on your machine and run `make lint` you'll encounter the following error:
```
--> Running linter
# honnef.co/go/tools/unused
../../../go/pkg/mod/honnef.co/go/tools@v0.4.3/unused/unused.go:419:14: obj.Origin undefined (type *types.Var has no field or method Origin)
../../../go/pkg/mod/honnef.co/go/tools@v0.4.3/unused/unused.go:421:14: obj.Origin undefined (type *types.Func has no field or method Origin)
note: module requires Go 1.19
# honnef.co/go/tools/staticcheck
../../../go/pkg/mod/honnef.co/go/tools@v0.4.3/staticcheck/lint.go:4025:29: righti.Method(i).Origin undefined (type *types.Func has no field or method Origin)
../../../go/pkg/mod/honnef.co/go/tools@v0.4.3/staticcheck/lint.go:4030:36: sel.Obj().(*types.Func).Origin undefined (type *types.Func has no field or method Origin)
note: module requires Go 1.19
make: *** [lint] Error 2
```

Reply:
I believe this is why @MSalopek had to update the linter workflow in #2324.